.storybook/themes/news.scss

```diff
@@ -1 +1,188 @@
+				),
+				right-rail-navigation: (
+					position: sticky,
+					top: 0,
+					width: 100%,
+					z-index: 9,
+					margin: 0 0 var(--global-spacing-5) 0,
+				),
+				right-rail-full-width-1: (
+					margin: 0 0 var(--global-spacing-5) 0,
+				),
+				right-rail-main: (
+					gap: var(--global-spacing-6),
+					max-width: calc(var(--content-max-width) * 1px),
+					width: var(--content-scale-width),
+					margin: auto,
+				),
+				right-rail-full-width-2: (
+					margin: var(--global-spacing-6) 0 0 0,
+				),
+				right-rail-rail-container: (
+					grid-template-columns: initial,
+					grid-template-rows: 1fr,
+					gap: var(--global-spacing-6),
+				),
+				right-rail-main-side-rail: (
+					padding: initial,
+				),
+				right-rail-main-interior-item: (
+					padding: initial,
+					border-right: initial,
+				),
+				right-rail-footer: (
+					margin: var(--global-spacing-10) 0 var(--global-spacing-8) 0,
+				),
+				right-rail-advanced: (
+					gap: var(--global-spacing-6),
+				),
+				right-rail-advanced-navigation: (
+					position: sticky,
+					top: 0,
+					width: 100%,
+					z-index: 9,
+					margin: 0 0 var(--global-spacing-5) 0,
+				),
+				right-rail-advanced-full-width-1: (
+					margin: 0 0 var(--global-spacing-5) 0,
+				),
+				right-rail-advanced-main: (
+					gap: var(--global-spacing-6),
+					max-width: calc(var(--content-max-width) * 1px),
+					width: var(--content-scale-width),
+					margin: auto,
+				),
+				right-rail-advanced-full-width-2: (
+					margin: var(--global-spacing-6) 0 0 0,
+				),
+				right-rail-advanced-rail-container: (
+					grid-template-areas: "main-side-rail-top" "main-interior-item-1" "main-side-rail-middle"
+						"main-interior-item-2" "main-side-rail-bottom",
+					grid-template-columns: initial,
+					grid-template-rows: 1fr,
+					grid-auto-rows: auto,
+					gap: var(--global-spacing-6),
+				),
+				right-rail-advanced-main-side-rail: (
+					padding: initial,
+					gap: var(--global-spacing-5),
+					display: contents,
+				),
+				right-rail-advanced-main-interior-item: (
+					padding: initial,
+					border-right: initial,
+					gap: var(--global-spacing-5),
+					display: contents,
+				),
+				right-rail-advanced-footer: (
+					margin: var(--global-spacing-10) 0 var(--global-spacing-8) 0,
+				),
+				right-rail-advanced-main-interior-item-1: (
+					grid-area: main-interior-item-1,
+				),
+				right-rail-advanced-main-interior-item-2: (
+					grid-area: main-interior-item-2,
+				),
+				right-rail-advanced-main-side-rail-top: (
+					grid-area: main-side-rail-top,
+				),
+				right-rail-advanced-main-side-rail-middle: (
+					grid-area: main-side-rail-middle,
+				),
+				right-rail-advanced-main-side-rail-bottom: (
+					grid-area: main-side-rail-bottom,
+				),
+			),
+		),
+		desktop: (
+			blocks: (
+				right-rail-navigation: (
+					padding: 0 0 var(--global-spacing-8) 0,
+				),
+				right-rail-rail-container: (
+					grid-template-columns: 2fr 1fr,
+					gap: initial,
```

Comment:
did you want `initial` or `0`?  `initial` is `normal` and normal is browser specific and has special rules: https://developer.mozilla.org/en-US/docs/Web/CSS/column-gap#syntax

Reply:
will update

Reply:
updated -- thanks @nschubach